plugins/task-plugin/src/che-workspace-client.ts

```diff
@@ -45,7 +45,7 @@ export class CheWorkspaceClient {
 
   async getMachineExecServerURL(): Promise<string> {
     const machineExecEndpoint = await this.getMachineExecEndpoint();
-    const port = machineExecEndpoint.attributes?.port || machineExecEndpoint.attributes?.targetPort;
+    const port = machineExecEndpoint.attributes?.port || machineExecEndpoint.targetPort;
```

Comment:
from what I understand, if you map in `CheServerEndpointServiceImpl` the property targetPort
then here we could just use `machineExecEndpoint.targetPort` all the time no ?

Reply:
Yeap, it's here from the time I went another way.
I'll try to remove it